samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/About.razor.css

```diff
@@ -49,6 +49,16 @@
     font: normal normal normal 12px/18px 'Helvetica Neue', Arial, sans-serif;
 }
 
+.pro {
+    color: gold;
+    font-style: italic;
+    font-weight: bold;
+    padding: 1rem 0;
+}
+
+.pro a {
+    color: gold!important;
```

Comment:
just poking in to say generally speaking, it's best not to use !important most of the time in CSS. Specifying the class and element like you did here should be enough to ensure no other rules in this file take priority. Is there maybe another reason !important is needed?

Reply:
@agnesstelmach you are absolutely correct, and it was a shortcut months ago that I used instead of taking the time to figure out why it didn't work without it. I will try to revisit soon, or feel free when you're working in GB to update!